presto-main/src/main/java/com/facebook/presto/operator/PartitionedOutputOperator.java

```diff
@@ -317,6 +332,7 @@ public PagePartitioner(
             this.partitionConstants = requireNonNull(partitionConstants, "partitionConstants is null").stream()
                     .map(constant -> constant.map(NullableValue::asBlock))
                     .collect(toImmutableList());
+            this.replicateAny = requireNonNull(replication, "replication is null") == Replication.REPLICATE_NULLS_AND_ANY;
```

Comment:
I'm not convinced this is worth it. It's single use place and the actual enum constant meaning is very... this-place-specific.